com/haxepunk/HXP.hx

```diff
@@ -183,6 +185,17 @@ class HXP
 	public static var orientations:Array<Int> = [];
 
 	/**
+	 * Custom cursor used to update and render last on scene
+	 */
+	public static var cursor(default, set):Cursor;
+	private static inline function set_cursor(cursor:Cursor):Cursor
+	{
+		if (cursor == null) Mouse.hide();
```

Comment:
Looks backwards. If I set the custom cursor to null, it will also hide the native cursor.